test/models/erratum_test.rb

```diff
@@ -168,16 +168,17 @@ def test_installable_for_hosts
       refute_includes errata, @enhancement
     end
 
-    def test_installable_for_hosts_dev_environment
-      errata = Erratum.installable_for_hosts([@host_dev, @host_without_errata])
-      assert_includes errata, @security
-      assert_includes errata, @bugfix
-      refute_includes errata, @enhancement
+    def test_installable_for_hosts_with_no_bound_repos
+      # make sure the @host_dev has no bound repositories
```

Comment:
can you fix this comment to say @host ?  Also @host_dev is no longer used, can you remove it from the setup?